src/components/Button/Button.scss

```diff
@@ -26,7 +23,8 @@
     }
   }
 
-  &:hover {
+  &:hover:not(.disable) {
```

Comment:
maybe we might want to check with the design team. when we hover our mouse cursor over the disabled button, should it show a different mouse cursor?

Reply:
i see that the cursor becomes disabled